docs/networks.md

```diff
@@ -3,14 +3,13 @@
 
 Networks in the HaaS have three fields:
 
-- 'creator' represents what project created it; or 'admin' if it was
-  administrator-created.  If there are still networks created by a project,
-  then that project cannot be deleted.  Only the creator has the ability to
+- 'owner' represents what project owns it; or 'admin' if it was
```

Comment:
If we're going to keep "admin" as a special project name, we should either a) stick a restriction in `project_create` and document or b) make it an illegal project name (like <admin>), though this would only apply once #600 is implemented.